apps/app/.env.development

```diff
@@ -1,5 +1,6 @@
 NEXT_PUBLIC_NAME=DEVELOPMENT
-NEXT_PUBLIC_API_URL=https://localhost:1337/api
-NEXT_PUBLIC_STRAPI_URL=https://localhost:1337/api
-NEXT_PUBLIC_GOOGLE_AUTH_URL=https://localhost:1337/connect/google
+NEXT_PUBLIC_API_BASE_URL=http://localhost:1337
+NEXT_PUBLIC_API_URL=http://localhost:1337/api
```

Comment:
Do we need both this and `NEXT_PUBLIC_STRAPI_URL`? They have the same value.

Reply:
Base url has been changed accordingly